### DISCUSSION TITLE
How to remove header from drawer.navigator

### DISCUSSION THREAD

**Discussion by komalharmale** [react-navigation/react-navigation#10091] [General]
![Screenshot 2021-11-02 at 2 31 12 PM](https://user-images.githubusercontent.com/56529640/139816937-be67267a-a2f8-423e-9ece-6d5d24eb0dbf.png)
hi i am trying to remove header shown on drawer.navigator i have trying different solution but its not working
<Drawer.Navigator screenOptions={{drawerPosition: 'right', swipeEdgeWidth: 0}}drawerContent={props => <CustomDrawerContent {...props} />}> <Drawer.Screen name="Homescreencomponent" options={{headerShown: false}} component={Homescreencomponent} /> </Drawer.Navigator>

**Comment by raajnadar** [comment]
You can check those small things in the documentation https://reactnavigation.org/docs/drawer-navigator#headershown

Make sure you are using the `headerShown` in the right navigator, because in drawer `headerShow` is already false.

P.S. - In the image it seems like it is not react navigation header.
